gtest/src/program.rs

```diff
@@ -207,16 +210,95 @@ impl<'a> Program<'a> {
             .join(path)
             .clean();
 
-        let program_id = id.clone().into().0;
+        let filename = path.file_name().and_then(OsStr::to_str).unwrap_or_default();
+        assert!(
+            !filename.ends_with(".meta.wasm"),
+            "Cannot load `.meta.wasm` file without `.opt.wasm` one. \
+            Use Program::from_opt_and_meta() instead"
+        );
+        let is_opt = filename.ends_with(".opt.wasm");
```

Comment:
what if I pass path not to .wasm file?

Reply:
Do you want I just assert file extension?

Reply:
yeah